<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClaudeConfigPage</class>
 <widget class="QWidget" name="ClaudeConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Claude Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>API Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>API Key:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="apiKeyEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Enter your Claude API key</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Model:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="modelComboBox">
        <item>
         <property name="text">
          <string>claude-3-opus-20240229</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>claude-3-sonnet-20240229</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>claude-3-haiku-20240307</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>claude-3.5-sonnet-20240620</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="autoSelectCheckBox">
        <property name="text">
         <string>Automatically send selected text</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="insertResponseCheckBox">
        <property name="text">
         <string>Insert response at cursor position</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>